examples/guided.py

```diff
@@ -190,7 +191,7 @@ def ask_user_questions():
 	# Additional packages (with some light weight error handling for invalid package names)
 	while True:
 		if not archinstall.arguments.get('packages', None):
-			print("Only packages such as base, base-devel, linux, linux-firmware, efibootmgr and optional profile packages are installed.")
+			print("Only packages such as base, base-devel, linux, linux-firmware, efibootmgr(on uefi systems)/grub(on bios systems) and optional profile packages are installed.")
```

Comment:
add a space before each parenthesis and make UEFI and BIOS caps.

Reply:
Actually, maybe do "efibootmgr (or GRUB)" if GRUB will be a choice on both.

Reply:
sure

Reply:
we only have efibootmgr on UEFI systems and GRUB by default on BIOS systems really speaking it isn't part of the base package I'm going with your first suggestion for now